webui/react/src/pages/TrialDetails/Header/TrialHeaderLeft.tsx

```diff
@@ -26,6 +32,21 @@ const TrialHeaderLeft: React.FC<Props> = ({ experiment, trial }: Props) => {
       <div className={css.trial}>
         <ExperimentIcons state={trial.state} />
         <div>Trial {trial.id}</div>
+        {trial.logSignal &&
+          (trial.logSignal.length < labelMaxLength ? (
+            <Badge
+              backgroundColor={hex2hsl(labelColor)}
+              key={trial.logSignal}
+              text={trial.logSignal}
+            />
+          ) : (
+            <Tooltip content={trial.logSignal} key={trial.logSignal}>
```

Comment:
this might show empty string based on the condition. is it ok?

Reply:
I might be missing something, could you please explain why would there be empty string?

Reply:
```
{trial.logSignal &&
          (trial.logSignal.length < labelMaxLength
```
i missed the second condition. nvm. its a nitpick but might be more readable if these 2 conditions are in the same line if formatter allows us